content/docs/hardware/v00.03.12/parts/_index.en.md

```diff
@@ -0,0 +1,235 @@
+</tr>
+<tr>
+  <td>1</td>
+  <td>Resistance (THT) 300 kΩ, 250 mW, 5%</td>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-300-kohm-0207-250-mw-5--1-4w-300k-p1407.html">Link</a></td>
+</tr>
+<tr>
+  <td>2</td>
+  <td>Resistance (THT) 1.1 kΩ, 250 mW, 5 %</td>
+  <td><a href="https://www.reichelt.de/widerstand-kohleschicht-1-1-kohm-0207-250-mw-5--1-4w-1-1k-p1318.html?&trstct=pos_0&nbc=1">Link</a></td>
+</tr>
+<tr>
+  <td>1</td>
+  <td>Ceramic capacitor 100 nF, 10%</td>
+  <td><a href="https://www.reichelt.de/vielschicht-keramikkondensator-100n-10--x7r-2-5-100n-p22853.html">Link</a></td>
+</tr>
+<tr>
+  <td>2</td>
+  <td>Electrolytic capacitor, radial, 22 µF, 10 V, low ESR</td>
+  <td><a href="https://www.reichelt.de/elko-radial-22-uf-10-v-1000-h-low-esr-aec-q200-rad-fc-22-10-p84587.html">Link</a></td>
+</tr>
+<tr>
+  <td>1</td>
+  <td>Schottky diode, 60 V, 5 A, DO-201AD</td>
+  <td><a href="https://www.reichelt.de/schottkydiode-60-v-5-a-do-201ad-sb-560-p16081.html">Link</a></td>
+</tr>
+<tr>
+  <td>1</td>
+  <td>Resetting fuse, max. 40 A - 30 V, 6.6 sec<br>Type PFRA, radial design</td>
+  <td><a href="https://www.reichelt.de/rueckstellende-sicherungen-max-40a-30v-6-6s-pfra-110-p35211.html">Link</a></td>
+</tr>
+
+
+<tr><th colspan="3">Mechanical components</th></tr>
+<tr>
+  <td>4</td>
+  <td>Neodymium magnets, 20x10x2mm</td>
+  <td><a href="https://www.amazon.de/dp/B085CBZTQJ">Link</a></td>
+</tr>
+<tr>
+  <td>4</td>
+  <td>Neodymium magnets, 3x3x3mm</td>
+  <td><a href="https://www.amazon.de/dp/B079KDYBZ8">Link</a></td>
+</tr>
+<tr>
+  <td>28</td>
+  <td>M3x8mm socket head cap screw (DIN912). (1x GPS, 8x for two housing openings, 3x circuit board, 5x cover, 3x display, 6x 2 luggage rack brackets, 2x for cable strain relief), depending on the variant, fewer screws may be enough.</td>
+  <td></td>
+</tr>
+<tr>
+  <td>1</td>
+  <td>M3x30mm socket head cap screw (DIN912) (for the locking pin)</td>
+  <td></td>
+</td>
+<tr>
+  <td>10</td>
+  <td>M3 nut (DIN934) (4x each for two housing openings, 2x for cable strain relief, depending on the variant, fewer nuts may be enough.)</td>
+  <td></td>
+</tr>
+<tr>
+  <td>18</td>
+  <td>Thread insert / insert nut M3x5.7 - (1x GPS, 5x cover, 3x board, 3x display, 6x for 2 luggage rack mounts, depending on the variant, fewer nuts may be enough.)</td>
+  <td><a href="https://www.amazon.de/dp/B08BCRZZS3">Link1</a> <a href="https://turmberg3d.de/products/gewindeeinsatze-fur-kunststoffteile?variant=39376894066883">Link2</a></td>
+</tr>
+
+<tr><th colspan="3">Special designs</th></tr>
+<tr>
+  <td>1</td>
+  <td>PCB v00.03.12</td>
+  <td><a href="https://github.com/openbikesensor/OpenBikeSensor_PCB_Board/tree/merged/OpenBikeSensor03">Link</a></td>
+</tr>
+<tr>
+  <td>1</td>
+  <td>Main case, display case, bracket(s)</td>
+  <td><a href={< relref "docs/hardware/general/case" >}>Link</a></td>
+</tr>
+
+</tbody>
+</table>
```

Comment:
I think this section is rather difficult to maintain already, managing the links is already pretty finicky. Unless we go some data driven approach and group the links with labels in different languages in the same source file and then generate the table from that, we should not burden ourselves with two languages. Again, this is something that can be referenced from here and just link to the original. I don't think English-only speakers will have a hard time ordering the parts, as the part names are either already in English, obvious to translate, or link to a page that has a translation anyway. The notes above are important, but the table isn't.

Reply:
That's a fair point about the links. What I want to try and avoid is taking people out of the English side. The navigation changes when you change languages, and it's not always easy to make your way back. Perhaps adding a link on the German side back to the English building page might be a good solution. I'll make some edits and see what I can do.